code/game/turfs/open/space/transit.dm

```diff
@@ -36,13 +36,13 @@
 /turf/open/space/transit/proc/initialize_drifting(atom/entered, atom/movable/enterer)
 	SIGNAL_HANDLER
 
-	if(enterer && !HAS_TRAIT(enterer, TRAIT_HYPERSPACED))
+	if(enterer && !HAS_TRAIT(enterer, TRAIT_HYPERSPACED) && !(locate(/obj/structure/lattice) in contents))
```

Comment:
Can we make `lattice` apply a trait to the turf instead? I think we have an element or component for this.

Reply:
Yes! I changed the istypes to use that element with TRAIT_HYPERSPACE_STOPPED